server/game/cards/06-WoE/Cleric.js

```diff
@@ -0,0 +1,20 @@
+const Card = require('../../Card.js');
+
+class Cleric extends Card {
+    // When cleric enters play, capture 1A
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardEntersPlay: (event, context) => event.card === context.source
+            },
+            gameAction: ability.actions.capture((context) => ({
+                amount: 1,
+                target: context.event.card
```

Comment:
I think the amber can be captured on any creature, not necessarily Cleric

Reply:
That would be cool if true! Is there a source for that? I thought creatures that instruct you to capture would always capture onto themselves.

![image](https://github.com/keyteki/keyteki/assets/732758/58298eb3-9840-44d2-890a-3ee8b5f4e596)

Reply:
Yeah I agree that it has to capture on the new Cleric.  Otherwise it would say "on a friendly creature" or something.